packages/headless/tsconfig.json

```diff
@@ -8,5 +8,6 @@
     "esModuleInterop": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true
-  }
+  },
+  "exclude": ["./temp/**", "./dist/**", "./docs/**"]
```

Comment:
This change is small and unrelated to the PR, but I [mentioned I would include it](https://coveo.slack.com/archives/G7HD9P48H/p1613159452252200).